chainweb.openapi.yaml

```diff
@@ -2699,6 +2726,56 @@ paths:
               example:
                 $ref: "#/components/examples/nodeConfig"
 
+  /make-backup:
+    post:
+      summary: Start a backup job
```

Comment:
I think, it would be good to add a description that explains the usage:

* how are backups enabled in the node?
* how is it configured (rocksDb backup should be on same device and file system)
* where is the backup stored and what is the naming scheme?
* how much disk space is needed?
* how long will it take roughly?
* maybe mention that it is up to the user to cleanup old backups.